.pylintrc

```diff
@@ -545,7 +545,7 @@ spelling-dict=en_US
 spelling-ignore-comment-directives=fmt: on,fmt: off,noqa:,noqa,nosec,isort:skip,mypy:
 
 # List of comma separated words that should not be checked.
-spelling-ignore-words=
+spelling-ignore-words=fixme
```

Comment:
We already have a mechanism to mark words as 'correct' for spell check by adding them to this file. https://github.com/NEMRI-org/primo-optimizer/blob/main/.github/wordlist.txt

Please undo the change to pylintrc and instead add fixme to that file in alphabetical order

Reply:
Fixed